02-Dev-Cluster-Setup/scripts/startup.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 ##################################
-# replace with your ID (optinonal)
+# replace with your ID (optional)
```

Comment:
in the latest script, we changed this to sed with $USER

we should update the az command, the comments and the SSH commands (remove akdc@)